src/helpers.js

```diff
@@ -32,6 +33,28 @@ function reduceError (error = {}) {
   return error
 }
 
+/**
+ * Fetch a URL, with retry options provided or default retry options otherwise
+ * By default retries will happen for 14 seconds (3 retries at 1, 2 and then 4 seconds -- there cannot be enough time for anotehr retry after that)
+ * Retry will occur if error code 429 or >= 500 occurs.
+ *
+ * @param {string} url to fetch
+ * @param {*} options Fetch options object, which can also include retryOptions described here https://github.com/adobe/node-fetch-retry
+ * @returns {object} json response of calling fetch
+ */
+function nodeFetchRetry (url, options) {
+  const retryOptions = {
+    retryInitialDelay: 1000,
+    retryOnHttpResponse: response => (response.status >= 500) || response.status === 429,
+    retryMaxDuration: 14000
```

Comment:
why 14 seconds?

Reply:
First retry @ 1 sec, second at 2 sec, third at 4.  After the first three retries we have spent at least 7 seconds at a minimum. The next retry would require 8 seconds wait first, and therefore it will not retry again.  Requirements for the story were retries after 1,2, and 4 seconds.  So this number caps the number of retries at 3 based on all of that.